src/Mutant/MetricsCalculator.php

```diff
@@ -242,6 +255,14 @@ public function getCoveredCodeMutationScoreIndicator(): float
         return $this->getCalculator()->getCoveredCodeMutationScoreIndicator();
     }
 
+    /**
+     * Mutation Score Indicator relative to the total lines code coverage
+     */
+    public function getRelativeCoveredCodeMutationScoreIndicator(): float
+    {
+        return $this->getCalculator()->getRelativeCoveredCodeMutationScoreIndicator();
+    }
```

Comment:
Is this correct that this isn't used anywhere yet?

Is it going to be reported as a separate metric? It's okay to hide another in this case.